docs/reference/notices.md

```diff
@@ -0,0 +1,9 @@
+---
+id: notices
+title: "Notices"
+---
+### Security Notices
```

Comment:
What about general announcements? We do this for Camunda platform: https://docs.camunda.org/enterprise/announcement/

Reply:
I'm a bit torn about how to address this. It looks like half the content is announcing new supported features and frameworks and the other half is deprecation announcements. Is that how it will work for Camunda Cloud? 

I'd like to move https://docs.camunda.io/docs/product-manuals/zeebe/open-source/deprecated-features out of Zeebe specifically and rename the page from "deprecated features" to "announcements". Do we have any new items we need to list for GA?

Reply:
I assume for Cloud we will handle it in a similar way for now. 
For 1.0 topics @menski  probably knows best what environments we do not support anymore. Right now Elastic 6 comes to my mind. Elastic 7 is the new thing

Reply:
@menski Do we need to include Elastic 6 as an announcement for removing support? Or is your migration guide moving from 0.26 to 1.0 going to cover this?

Reply:
For Camunda platform (including Optimize) we are announcing it every time we do not support a specific database anymore. So if 6 is not supported anymore I suggest we announce it.

Reply:
@felix-mueller I'm going to resolve these comments and start a new thread. I have an idea 💡